requirements/admin-dev.txt.in

```diff
@@ -0,0 +1,26 @@
+# Run the command again, but include the test packages and omit any packages
```

Comment:
Would you mind either adding a README in this directory or something in our dev docs that defines what each of these requirements files are supposed to contain (and which ones should be supersets of other ones, etc.)?